packages/fast-components-styles-msft/src/patterns/button.ts

```diff
@@ -1,13 +1,17 @@
+import { focusOutlineWidth, outlineWidth } from "../utilities/design-system";
 import { ComponentStyles, CSSRules } from "@microsoft/fast-jss-manager";
-import { DesignSystem, ensureDesignSystemDefaults } from "../design-system";
 import { directionSwitch, format, toPx } from "@microsoft/fast-jss-utilities";
 import { applyCornerRadius, applyFocusPlaceholderBorder } from "../utilities/border";
 import { applyCursorPointer } from "../utilities/cursor";
-import { focusOutlineWidth, outlineWidth } from "../utilities/design-system";
+import { DesignSystem, ensureDesignSystemDefaults } from "../design-system";
 import { applyScaledTypeRamp } from "../utilities/typography";
 import { glyphSize, height, horizontalSpacing } from "../utilities/density";
 import { applyDisabledState } from "../utilities/disabled";
 import { ButtonBaseClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import {
+    highContrastDisabledBorder,
+    highContrastDisabledForeground,
+} from "../utilities/high-contrast";
```

Comment:
it does not appear we are using any of these here, can you revert the changes to this file?

Reply:
reverted